src/styles/blocks/card.scss

```
@@ -0,0 +1,78 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: $card-width;
+  height: $card-height;
+  padding: $card-padding;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 16px auto 0;
```

increase card padding-top instead
```suggestion
    margin: 0 auto;
```